client/my-sites/plans/main.jsx

```diff
@@ -293,6 +293,16 @@ class Plans extends Component {
 		return interval;
 	}
 
+	getIntervalForBusinessMigrationPlans() {
+		const { intervalType } = this.props;
+
+		// Only accept monthly or yearly for the interval; otherwise let the component provide a default.
+		const interval =
+			intervalType === 'monthly' || intervalType === 'yearly' ? intervalType : undefined;
+
+		return interval;
+	}
```

Comment:
Since we only allow yearly here and this interval is not used in `BusinessTrialPlansPage` component, maybe we don't need this ?